docs/core/diagnostics/dotnet-trace.md

```diff
@@ -3,17 +3,17 @@ title: dotnet-trace - .NET Core
 description: Installing and using the dotnet-trace command-line tool.
 author: sdmaclea
 ms.author: stmaclea
-ms.date: 10/14/2019
+ms.date: 10/28/2019
 ---
-# Trace for performance analysis utility (`dotnet-trace`)
+# dotnet-trace performance analysis utility
 
 **This article applies to:** .NET Core 3.0 SDK and later versions
 
-## Installing `dotnet-trace`
+## Install dotnet-trace
 
-To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+Install `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
 
-```dotnetcli
+```console
 dotnet tool install --global dotnet-trace
```

Comment:
@sdmaclea folks can now install this as a local tool as well right? Should we say that this is an example of how to install?

Reply:
Why are we changing this from `dotnetcli` to `console`?  This is a dotnet command.

Yes.   I don't see any reason why they couldn't install as a local tool. I haven't tried..

Reply:
Good call on the id change. It should continue to be dotnetcli.

Reply:
I think the only difference is the scope of the tool: global vs. current project/solution/repo.

Reply:
@mairaw Sounds like you want a text change. Like you suggested, there is no reason we have to recommend a global tool install here.

Reply:
Let's address this later.